src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/LSPDocumentSnapshot.cs

```diff
@@ -17,7 +17,7 @@ public abstract class LSPDocumentSnapshot
 
         public abstract IReadOnlyList<VirtualDocumentSnapshot> VirtualDocuments { get; }
 
-        public bool TryGetVirtualDocument<TVirtualDocument>(out TVirtualDocument virtualDocument) where TVirtualDocument : VirtualDocumentSnapshot
+        public virtual bool TryGetVirtualDocument<TVirtualDocument>(out TVirtualDocument virtualDocument) where TVirtualDocument : VirtualDocumentSnapshot
```

Comment:
\+ `virtual` to facilitate mocking.

Reply:
Is https://github.com/dotnet/aspnetcore-tooling/blob/master/src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test.Common/TestDocumentSnapshot.cs not sufficient? If not make sure to leave a comment here.

Reply:
Ahh it totally is, thanks!